packaging/crio.conf.d/microshift_amd64.conf

```diff
@@ -15,3 +15,4 @@ plugin_dirs = [
 # for community builds on top of OKD, this setting has no effect
 [crio.image]
 global_auth_file="/etc/crio/openshift-pull-secret"
+pause_image = "registry.k8s.io/pause:3.6"
```

Comment:
Shouldn't we add the OCP image here already?

Reply:
fixed.